### PR TITLE
Adding mysqlCnfOptions settings.

### PR DESCRIPTION
**Pull request by sk33lz**

### CODE REVIEW

File: lib/plugins/TaskRunner/LAMPApp.js [hunk 1]
```diff
@@ -133,22 +138,38 @@ class LAMPApp extends Script {
     }
   }
 
-  // a meta-script that installs all apache/php-related scripts
-  addScriptApachePhp() {
+  // a meta-script that installs all apache/mysql/php related scripts.
```
Comment (dzink) [inline lib/plugins/TaskRunner/LAMPApp.js]: Why is this piggy backing on the apache config? Code looks good here but each function should only do one thing; this should be made into its own function.

File: lib/plugins/TaskRunner/LAMPApp.js [hunk 2]
```diff
@@ -133,22 +138,38 @@ class LAMPApp extends Script {
...
-  // a meta-script that runs all setup commands
+  // a meta-script that runs all setup commands.
   addScriptSetup() {
     this.addScriptHeader();
     this.addScriptCliDefines();
     this.addScriptInstallPackages();
   }
 
+  addScriptMysqlCnfOptions() {
+    MYSQLCNF_PATH="/etc/mysql/conf.d"
```
Comment (dzink) [inline lib/plugins/TaskRunner/LAMPApp.js]: This constant should be moved to the top of the file.

Comment (dzink) [reply lib/plugins/TaskRunner/LAMPApp.js]: This function should also be near the addScriptRestartMysql file.

File: lib/plugins/TaskRunner/LAMPApp.js (hunk 1 above)
Comment (sk33lz) [reply lib/plugins/TaskRunner/LAMPApp.js]: Fixed.

File: lib/plugins/TaskRunner/LAMPApp.js (hunk 2 above)
Comment (sk33lz) [reply lib/plugins/TaskRunner/LAMPApp.js]: I've got the mysql functions in a separate function and moved the constant to the constants.js file with the rest of the constants.

**Comment by sk33lz** [comment]
@dzink please review this again when you have time. I have changed it up a bit to set a custom file that is included at the end of the my.cnf file instead of how I was doing it previously due to bugs I ran into with different versions of mysql we support in different images. This way it just appends the custom file and then we add configuration options to that file instead of trying to figure out what conf.d dir it needs to call from.

**Pull request merged**